requirements.txt

```diff
@@ -1,5 +1,4 @@
+incremental
 twisted
```

Comment:
What does incremental do?

Reply:
With all honesty, I don't know. All that I know that pip3 on fresh Mac failed to install twisted, complaining about `incremental`. It was super happy after installing it explicit before twisted!!

This is not perfect, but I feel the urge to have the solution working for Mac. So I opted to investigate later.